spec/support/landing_folder_support/file_objects/et3_pdf_file.rb

```diff
@@ -6,21 +6,22 @@ module FileObjects
       class Et3PdfFile < Base # rubocop:disable Metrics/ClassLength
         include RSpec::Matchers
 
-        def initalize(tempfile)
-          self.tempfile = tempfile
+        def initialize(*args, template:)
+          super(*args)
+          self.template = template
         end
 
         def has_correct_contents_for?(response:, respondent:, representative:, errors: [], indent: 1) # rubocop:disable Naming/PredicateName
-          has_header_for?(response, errors: errors, indent: indent) &&
-            has_claimant_for?(response, errors: errors, indent: indent) &&
-            has_respondent_for?(respondent, errors: errors, indent: indent) &&
-            has_acas_for?(response, errors: errors, indent: indent) &&
-            has_employment_details_for?(response, errors: errors, indent: indent) &&
-            has_earnings_for?(response, errors: errors, indent: indent) &&
-            has_response_for?(response, errors: errors, indent: indent) &&
-            has_contract_claim_for?(response, errors: errors, indent: indent) &&
-            has_representative_for?(representative, errors: errors, indent: indent) &&
-            has_disability_for?(respondent, errors: errors, indent: indent)
+            Et3PdfFileSection::HeaderSection.new(tempfile, template: template).has_contents_for?(response: response)
```

Comment:
This has now been changed to use a separate class for matching each section as there are alot of fields !